addons/portal/views/portal_templates.xml

```diff
@@ -302,9 +302,9 @@
                             <button type="button" class="btn btn-secondary dropdown-toggle" data-toggle="dropdown"/>
                             <div class="dropdown-menu" role="menu">
                                 <t t-foreach='searchbar_inputs' t-as='input'>
-                                    <a t-att-href="'#' + searchbar_inputs[input]['input']"
-                                        t-attf-class="dropdown-item#{search_in == searchbar_inputs[input]['input'] and ' active' or ''}">
-                                        <span t-raw="searchbar_inputs[input]['label']"/>
+                                    <a t-att-href="'#' + input_value['input']"
```

Comment:
where does `input_value` come from?

Reply:
`t-foreach` creates a bunch of in-scope names based on the `t-as`.

When iterating a mapping, the key goes into `{as}` and the value goes into `{as}_value` (for non-mapping, both get the same).